src/components/views/donate/EstimatedMatchingToast.tsx

```diff
@@ -45,54 +46,83 @@ const EstimatedMatchingToast = ({
 		allocatedFundUSDPreferred,
 		allocatedFundUSD,
 		allocatedTokenSymbol,
+		minimumValidUsdValue,
+		maximumReward,
 	} = activeStartedRound || {};
 
 	const esMatching = calculateEstimatedMatchingWithDonationAmount(
 		(tokenPrice || 0) * (amountTyped || 0),
 		projectDonationsSqrtRootSum,
 		allProjectsSum,
 		allocatedFundUSDPreferred ? allocatedFundUSD : matchingPool,
-		activeStartedRound?.maximumReward,
+		maximumReward,
 	);
 
+	const isAboveMinValidUsdValue =
+		minimumValidUsdValue != null
+			? amountTyped && amountTyped >= minimumValidUsdValue
```

Comment:
You have compared amountTyped with minimumValidUsdValue that's wrong, because first one is token amount like 0.01 ETH but second one is USD, so first you should fetch token price.
you can use `useTokenPrice`.

Reply:
You're right, thanks